apache-rat-core/src/main/java/org/apache/rat/Defaults.java

```diff
@@ -152,27 +163,35 @@ public SortedSet<ILicenseFamily> getLicenseFamilies(LicenseFilter filter) {
      * @param filter define which type of licenses to return.
      * @return The sorted set of approved licenseIds.
      */
-    public SortedSet<String> getLicenseIds(LicenseFilter filter) {
+    public SortedSet<String> getLicenseIds(final LicenseFilter filter) {
         return setFactory.getLicenseFamilyIds(filter);
     }
+
+    public static FilenameFilter getFilesToIgnore() {
+        return FILES_TO_IGNORE;
+    }
+
+    public static IOFileFilter getDirectoriesToIgnore() {
+        return DIRECTORIES_TO_IGNORE;
+    }
     
     /**
      * The Defaults builder.
      */
-    public static class Builder {
+    public final static class Builder {
         private final Set<URL> fileNames = new TreeSet<>(Comparator.comparing(URL::toString));
 
         private Builder() {
             fileNames.add(DEFAULT_CONFIG_URL);
         }
 
         /**
-         * Adds a URL to a configuration file to be read.
+         * Adds a URI to a configuration file to be read.
```

Comment:
URI or URL? Spotbugs complains that URL-comparison may lead to performance issues as each URL is resolved/queried via DNS ..... simply changing to URI is only psssible with more changes within RAT.

Reply:
Thanks.